### PR TITLE
Fix Unpkg CDN errors

### PR DESCRIPTION
**Pull request by williamhgough**

### CODE REVIEW

File: generators/app/templates/index.html [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
         </script>
-        <script data-main="App" src="https://unpkg.co/requirejs@2.3.3/require.js"></script>
+        <script data-main="App" src="https://unpkg.com/requirejs@2.3.3/require.js"></script>
         <script src="view/frontend/requirejs-config.js"></script>
         <script>
             (function (require) {
                 Object.assign(config.paths, {
-                    'react': '//unpkg.com/react/dist/react',
-                    'react-dom': '//unpkg.com/react-dom/dist/react-dom',
+                    'react': '//unpkg.com/react@16.0.0/umd/react.production.min',
```
Comment (jakestubbz) [inline generators/app/templates/index.html]: Upgrading to React 16 might have some breaking changes for our applications, we need to be careful with this. If we merge this, we're going to have to go through and make sure everything still works as expected & refactor where necessary.

We do need to upgrade at some point but do we have time right now?

https://reactjs.org/blog/2017/09/26/react-v16.0.html

Comment (williamhgough) [reply generators/app/templates/index.html]: Want me to drop back to 15.4 for now?

**Comment by jakestubbz** [comment]
Is this PR still valid @williamhgough?

**Comment by williamhgough** [comment]
@jakestubbz yeah! going to update those links to `v16.1.0` though.

**Pull request merged**